js/registration.js

```diff
@@ -0,0 +1,117 @@
+const form = document.getElementById('form');
+const username = document.getElementById('username');
+const password = document.getElementById('password');
+const email = document.getElementById('email');
+const email2 = document.getElementById('email2');
+
+form.addEventListener('submit', e => {
+    e.preventDefault();
+
+    validateInputs();
+    if(validateInputs()) {
+         saveData();
+         const inputs = document.querySelectorAll('#username, #password, #email, #email2');
+          inputs.forEach(input => {
+          input.value = '';
+        });
+        window.location.href='profile.html';
+    }
+});
+
+const setError = (element, message) => {
+    const formControl = element.parentElement;
+    const errorDisplay = formControl.querySelector('small');
+
+    errorDisplay.innerText = message;
+    formControl.classList.add('error');
+    formControl.classList.remove('success')
```

Comment:
To sie tez ladnie nadaje na funkcje. np.
```js
const displayError = (errorDisplay, formControl, message) => {
    errorDisplay.innerText = message;
    formControl.classList.toggle('error');
    formControl.classList.toggle('success');
};
```
`toggle` sprawia, ze mozesz re-uzyc tej funkcji w `setSuccess` rowniez.

Reply:
Sprawdzałam i nie działa poprawnie (jak jest błąd, to wypisuje komunikat na czerwono, ale border na elemencie jest zielony)— moim zdaniem dlatego, że musi mieć ustawione coś na start, dlatego zostawiam `add` i `remove`